native-extension/klarna/checkout/klarna-checkout/README.md

```diff
@@ -0,0 +1,27 @@
+# KlarnaCheckout
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 11.2.14.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
```

Comment:
I believe these commands are from the Angular CLI. Do you want to replace them with `npm start`, `npm run build`, etc?

Reply:
This is a generated file which should be replaced by Shaun's readme